manifests/candlepin.pp

```diff
@@ -131,6 +126,14 @@
       key_mode    => '0440',
     }
 
+    keystore { $keystore:
+      ensure        => present,
```

Comment:
```suggestion
      ensure        => 'present',
```

NIT: I've previously received feedback that it's preferred to avoid naked strings

Reply:
There's some edge cases, like `present` and `installed` for `ensure`. I those are very common. I'm not sure we need to start converting those.